app/forms/AssessmentFormProvider.scala

```diff
@@ -19,22 +19,27 @@ package forms
 import forms.mappings.Mappings
 import models.AssessmentAnswer
 import play.api.data.Form
-
 import javax.inject.Inject
+import play.api.data.Forms.seq
 
 class AssessmentFormProvider @Inject() extends Mappings {
 
-  def apply(exemptionCount: Int): Form[AssessmentAnswer] = {
-    val messagesKey = if (exemptionCount == 1) { "assessment.error.required.onlyOne" }
-    else { "assessment.error.required" }
+  def apply(): Form[AssessmentAnswer] = {
+
+    val messagesKey = "assessment.error.required"
+
     Form(
       "value" ->
-        text(messagesKey)
+        seq(text(messagesKey))
+          .verifying(messagesKey, values => values.nonEmpty)
           .verifying(
             messagesKey,
-            value => Set("true", "false").contains(value)
+            values => !(values.contains("none") && values.size > 1)
```

Comment:
simply asserts that if 'none of the above' is selected, it is the only one selected. this enforced client-side, also.

Reply:
that's a nicer way of doing it :)